website/events/api/v2/admin/views.py

```diff
@@ -72,9 +72,17 @@ class EventRegistrationAdminListView(AdminListAPIView, AdminCreateAPIView):
     required_scopes = ["events:admin"]
     filter_backends = (
         framework_filters.OrderingFilter,
+        framework_filters.SearchFilter,
         filters.EventRegistrationCancelledFilter,
     )
     ordering_fields = ("queue_position", "date", "date_cancelled")
+    search_fields = (
+        "member__profile__nickname",
+        "member__first_name",
+        "member__last_name",
+        "member__username",
```

Comment:
the username is basically the first and last name combined, already have those

Reply:
For the member list api we search on all 4, which is why I added them all here. Would you prefer just first and last name?